requirements.txt

```diff
@@ -1,2 +1,3 @@
 pandas
 beautifulsoup4
+demjson
```

Comment:
It's unmaintained, should we really use this one?

Reply:
Either that or I have to go back to the regex code i wrote. But I can move it to `utils/json.py`: `from_js()`.